src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -1016,48 +1020,54 @@ static inline int MPIDI_NM_mpi_win_allocate_shared(MPI_Aint size,
                 shm_unlink(shm_key);
 
             MPIR_ERR_SETANDSTMT(mpi_errno, MPI_ERR_NO_MEM, goto fn_fail, "**nomem");
+            }
```

Comment:
It's very confusing to have the new `}` on one commit but the indentation adjustments for everything inside the `while` statement in the next commit.